### ISSUE TITLE
TypeError: wtf.plugin is not a function at https://observablehq.com/@spencermountain/wtf_wikipedia

### ISSUE THREAD

**Issue by matkoniecz** [spencermountain/wtf_wikipedia#408]
```
const wtf = require('wtf_wikipedia')
wtf.plugin(require('wtf-plugin-html'))
```

fails with `TypeError: wtf.plugin is not a function`

https://observablehq.com/@spencermountain/wtf_wikipedia

https://github.com/spencermountain/wtf_wikipedia#plugins seems to indicate that syntax for achieving this was changed

**Comment by spencermountain** [comment]
hey Mateusz, this works for me, which browser are you using?
cheers

**Comment by matkoniecz** [comment]
I copied it into a node script. Is it supposed to be run only in a browser?

**Comment by spencermountain** [comment]
agh, sorry. I think you'll want `wtf.extend()`
looks like the alias for .plugin is missing. Will add it to the next release.
thanks for your help!

**Comment by spencermountain** [comment]
fixed in 9.0.0
cheers
